src/mz/src/api.rs

```diff
@@ -70,6 +70,7 @@ pub struct Region {
 pub struct Environment {
     pub environmentd_pgwire_address: String,
     pub environmentd_https_address: String,
+    pub resolvable: bool,
```

Comment:
Does this get populated by our backend? (I'm not familiar with the mz codebase.)

Reply:
Yep, it is - the environment controller does a dns resolution of the name against all the authoritative NS servers & sets that to true once they all can resolve the name.